tests/x11/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright 2012-2019 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: add addon to SLES via SCC
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base qw(y2_installbase y2_module_guitest);
+use strict;
+use warnings;
+use testapi;
+use registration;
+use version_utils qw(is_sle is_leap);
+use x11utils 'turn_off_gnome_screensaver';
+use YaST::Module;
+use utils;
+
+sub test_setup {
+    select_console "x11";
+    x11_start_program('xterm');
+    turn_off_gnome_screensaver;
+}
+
+sub run {
+    test_setup;
+    YaST::Module::open(module => 'scc', ui => 'qt');
+    save_screenshot;
+    $testapi::distri->get_registration()->register_via_scc({
+            email => get_var('SCC_EMAIL'),
+            reg_code => get_var('SCC_REGCODE')});
+    save_screenshot;
+    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
+    save_screenshot;
+    #TODO handle reg code if needed
+}
```

Comment:
where are the rest of the modules and extensions to mimic what we have here? now we are using addons we should update the settings to register all of them like in the original case  https://openqa.suse.de/tests/9339213#step/suseconnect_scc/70 In the video I cannot see any registration. I guess we are clicking next but shouldn't be some popups afters https://openqa.suse.de/tests/9339213#step/suseconnect_scc/70 ?